templates/coderd.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     app.kubernetes.io/component: {{ include "coder.serviceName" . }}
     app: {{ include "coder.serviceName" . }}
     coder.deployment: {{ include "coder.serviceName" . }}
-  annotations: {{ toYaml .Values.services.annotations | nindent 4 }}
+  annotations: {{ toYaml (merge .Values.coderd.annotations .Values.services.annotations) | nindent 4 }}
```

Comment:
Do this to handle the null case:

```
  annotations: {{ merge (.Values.postgres.default.annotations | default (dict)) (.Values.services.annotations | default (dict)) | quote }}
```

should also add test cases for this, it makes the chart a bit more robust :)

Reply:
awesome - thank you for the help!